src/web/handlers.rs

```diff
@@ -24,34 +31,31 @@ pub async fn get_tiles(
     trace!("get_tiles");
     metrics.incr("tiles.update");
 
-    let cinfo = request.connection_info();
     let settings = &state.settings;
-    let ip_addr_str = cinfo.remote_addr().unwrap_or({
-        let default = state
-            .adm_country_ip_map
-            .get("US")
-            .expect("Invalid ADM_COUNTRY_IP_MAP settting");
-        if let Some(country) = &treq.country {
-            state.adm_country_ip_map.get(country).unwrap_or(default)
-        } else {
-            default
+    let mut addr = None;
+    if let Some(header) = request.headers().get(&*X_FORWARDED_FOR) {
+        if let Ok(value) = header.to_str() {
+            addr = value
+                .split(',')
+                .next()
```

Comment:
Nit: let's add a comment to clarify that the real client IP should be the first chunk of `X-Forwarded-For` in our system setup (GCLB -> Openresty (Real IP) -> Contile).